ebos/equil/initstateequil_impl.hh

```diff
@@ -846,18 +859,21 @@ accountForScaledSaturations()
 }
 
 template <class MaterialLawManager, class FluidSystem, class Region, typename CellID>
-double PhaseSaturations<MaterialLawManager, FluidSystem, Region, CellID>::
+std::tuple<double, bool>
+PhaseSaturations<MaterialLawManager, FluidSystem, Region, CellID>::
 applySwatInit(const double pcow)
 {
-    return this->applySwatInit(pcow, this->swatInit_[this->evalPt_.position->cell]);
+    auto [swout, newSwatInit] = this->applySwatInit(pcow, this->swatInit_[this->evalPt_.position->cell]);
+    return {swout, newSwatInit};
```

Comment:
Is there a reason you can't use the original implementation here?  You're just forwarding to a different function with the same return type.

Reply:
Think I got some errors here at some point and made the changes. But they are unnecessary I see now, so I reverted back to the original implementation.